### PR TITLE
Update deprecation notice

### PR DESCRIPTION
**Pull request by JensTrier**
The cmdlets and parameters have now been deprecated. Updated the notes.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @get-itips @tseward

File: skype/skype-ps/skype/Set-CsUser.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ Accept wildcard characters: False
 When set to True, enables a user's voice mail calls to be routed to a hosted version of Microsoft Exchange Server.
 In addition, setting this option to True enables Skype for Business users to directly place a call to another user's voice mail.
 
-**Note**: Using this parameter will be deprecated for Teams users and it is not required anymore to set it for Microsoft Teams users.
+**Note**: It is not required anymore to set this for Microsoft Teams users.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:178]:
```suggestion
**Note**: It is not required anymore to set this parameter for Microsoft Teams users.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested change. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested change. Many thanks.

Thanks Johan! Done

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

File: skype/skype-ps/skype/Set-CsUser.md [hunk 2]
```diff
@@ -175,7 +175,7 @@ Accept wildcard characters: False
 When set to True, enables a user's voice mail calls to be routed to a hosted version of Microsoft Exchange Server.
 In addition, setting this option to True enables Skype for Business users to directly place a call to another user's voice mail.
 
-**Note**: Using this parameter will be deprecated for Teams users and it is not required anymore to set it for Microsoft Teams users.
+**Note**: It is not required anymore to set this parameter for Microsoft Teams users.
```
Comment (tiburd) [inline skype/skype-ps/skype/Set-CsUser.md:178]:
```suggestion
**Note**: It is not required to set this parameter for Microsoft Teams users.
```

**Pull request merged**